lib/Models/CsvCatalogItem.js

```diff
@@ -419,14 +425,19 @@ function loadTableFromCsv(item, csvString) {
     tableStructure.loadFromCsv(csvString);
     setActiveTimeColumn(tableStructure, tableStyle);
     item._tableStructure = tableStructure;
+    // Does the csv have addresses we can translate to long and lat?
+    if (!tableStructure.hasLatitudeAndLongitude && tableStructure.hasAddress)
+    {
+        var addrConverter = new AddressConverter(tableStructure, item.terria.corsProxy);
```

Comment:
Ok IMO it's not ok for `CsvCatalogItem` to be hard-coded to resolving Australian addresses using GNAF.  This needs to be a plugin point somehow.